demo/summer-2016.html

```diff
@@ -234,11 +235,29 @@
         }
       });
     };
+
     editor.setOptions({
         enableBasicAutocompletion: true,
         enableSnippets: true,
-        enableLiveAutocompletion: false
+        enableLiveAutocompletion: true
     });
+   
+    var warmUpCompleter = {
+        getCompletions: function(editor, session, pos, prefix, callback) {
+            var wordList = ["\\begin{itemize} \n \n\\end{itemize}", "\\item"];
```

Comment:
is it possible to position caret between the tags after completion? Now it positions after closing \end{itemize} which is annoying.